test/functional/feature_bind_extra.py

```diff
@@ -37,37 +37,41 @@ def setup_network(self):
         loopback_ipv4 = addr_to_hex("127.0.0.1")
 
         # Start custom ports by reusing unused p2p ports
-        port = p2p_port(self.num_nodes)
+        def extra_port():
```

Comment:
```suggestion
        def new_port():
```
NIT; wouldn't `new_port()` be more suitable ? line 50 is just the use of a new port not an extra port. "extra to what?"